src/test/scala/io/github/spark_redshift_community/spark/redshift/RedshiftSourceSuite.scala

```diff
@@ -0,0 +1,598 @@
+/*
+ * Copyright 2015 TouchType Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.github.spark_redshift_community.spark.redshift
+
+import java.io.{ByteArrayInputStream, OutputStreamWriter}
+import java.net.URI
+
+import com.amazonaws.services.s3.model.{BucketLifecycleConfiguration, S3Object, S3ObjectInputStream}
+import com.amazonaws.services.s3.model.BucketLifecycleConfiguration.Rule
+import io.github.spark_redshift_community.spark.redshift.Parameters.MergedParameters
+import org.apache.http.client.methods.HttpRequestBase
+import org.mockito.Matchers._
+import org.mockito.Mockito
+import org.mockito.Mockito.when
+import org.apache.hadoop.fs.{FileSystem, Path}
+import org.mockito.invocation.InvocationOnMock
+import org.mockito.stubbing.Answer
+import org.scalatest.{BeforeAndAfterAll, BeforeAndAfterEach, Matchers}
+import org.apache.spark.SparkContext
+import org.apache.spark.sql.sources._
+import org.apache.spark.sql._
+import org.apache.spark.sql.types._
+import org.apache.spark.sql.catalyst.InternalRow
+import org.apache.spark.sql.catalyst.encoders.RowEncoder
+import com.amazonaws.services.s3.AmazonS3Client
```

Comment:
This line is in the original `RedshiftSourceSuite.scala` but appears in sorted order there; not sure why it got moved to the bottom.